src/styles/main.scss

```
@@ -1,3 +1,64 @@
 body {
-  margin: 0;
+margin: 0;
+font-size: 100px;
+font-family: Arial, Helvetica, sans-serif;
 }
+
+.container {
+display: grid;
+align-items: center;
+justify-content: center;
+flex-wrap: wrap;
+
+@media (min-width: 900px ) {
+  & {
+    grid-template-columns: repeat(3, 1fr) ;
+    grid-template-rows: repeat(2, 1fr) ;
+  }
+}
+
+@media (min-width: 300px ) and (max-width: 599px) {
+      & {
+        grid-template-columns: 1fr ;
+        grid-template-rows: repeat(6, 1fr);
+      }
```

The same as in the previous task,
```suggestion
        grid-template-columns: 1fr ;
        grid-template-rows: repeat(6, 1fr);
```